plugins/backstage-plugin-argo-cd/src/types.ts

```diff
@@ -10,7 +10,7 @@ export const argoCDAppDetails = t.type({
       status: t.string,
     }),
     operationState: t.type({
-      finishedAt: t.string,
+      startedAt: t.string,

```

Comment:
Even though I have nothing against adding startedAt property, I would keep finishedAt in a form of:
finishedAt: t.union([t.string, t.undefined])
(The reason behind this is we use finishedAt in order to display 'Last synced' data) https://github.com/RoadieHQ/roadie-backstage-plugins/blob/main/plugins/backstage-plugin-argo-cd/src/components/ArgoCDDetailsCard.tsx